Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -2100,8 +2100,6 @@ public async Task RunMetadataSavers(BaseItem item, ItemUpdateType updateReason)
             {
                 await ProviderManager.SaveMetadataAsync(item, updateReason).ConfigureAwait(false);
             }
-
-            await UpdateImagesAsync(item, updateReason >= ItemUpdateType.ImageUpdate).ConfigureAwait(false);
```

Comment:
Is it instead saving with `BaseItem.RefreshMetadata`?

Reply:
with the efcore refactor, images are always evaluated when saved on the UpdateOrInsert method in a transactional matter